src/transformers/models/auto/configuration_auto.py

```diff
@@ -247,29 +247,38 @@
 )
 
 
+def _get_class_name(model_class):
+    if isinstance(model_class, (list, tuple)):
+        return " or ".join([f":class:`~transformers.{c.__name__}`" for c in model_class])
+    return f":class:`~transformers.{model_class.__name__}`"
```

Comment:
This change and below are for the docstrings to properly render a model type with several models possible: 
![image](https://user-images.githubusercontent.com/35901082/114095300-41fe6800-988b-11eb-98fc-51e24282455e.png)

Reply:
oh this is very nice